.github/workflows/add-changelog-item.yml

```diff
@@ -0,0 +1,31 @@
+name: Add changelog item
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  add-changelog-item:
+    runs-on: ubuntu-latest
+    container: node:14
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: changelog
```

Comment:
Looks like the branch has node_modules committed. Just heads up.

Reply:
TY

Reply:
removed in the branch, thanks